shared/strings_util.h

```diff
@@ -4,6 +4,61 @@
 #include <string>
 #include <vector>
 #include <sstream>
+#include <fstream>
+
+inline std::string ToLower(std::string str) {
+  for (char &c : str) if (c >= 'A' && c <= 'Z') c += 'a' - 'A';
+  return str;
+}
+
+inline std::string Hyphenate(std::string snake) {
+  for (char &c : snake) if (c == '_') c = '-';
+  return snake;
+}
+
+inline std::string Spaceify(std::string snake) {
+  for (char &c : snake) if (c == '_') c = ' ';
+  return snake;
+}
+
+inline std::string Capitalize(std::string str) {
+  if (char &c = str[0]; c >= 'a' && c <= 'z') c -= 'a' - 'A';
+  return str;
+}
+
+inline bool ParseBool(const std::string &b) {
```

Comment:
should probably accept 0/1 here too

Reply:
Could also add a parameter for what should be returned if the string can't be parsed.